packages/ses/README.md

```diff
@@ -30,20 +30,25 @@ npm install ses
 ### Lockdown
 
 SES introduces the `lockdown()` function.
-Calling `lockdown()` alters the surrounding execution enviornment, or
+Calling `lockdown()` alters the surrounding execution environment, or
 **realm**, such that no two programs running in the same realm can observe or
 interfere with each other until they have been introduced.
 
-To this end, `lockdown()` freezes all objects accessible to any program in the
-realm.
+To do this, `lockdown()` tamper-proofs all of the JavaScript intrinsics, to
+prevent **prototype pollution**.
+After that, no program can subvert a method of one of these objects (mitigating
+some **man in the middle attacks**), and no program can use these mutable
+objects to pass notes to parties that haven't been expressly introduced
+(mitigating some **covert communication channels**).
+
+Lockdown freezes all objects accessible to any program in the realm.
 The set of accessible objects includes but is not limited to: `globalThis`,
 `[].__proto__`, `{}.__proto__`, `(() => {}).__proto__` `(async () =>
 {}).__proto__`, and the properties of any accessible object.
 
 The `lockdown()` function also **tames** some of those accessible objects
 that have powers that would otherwise allow programs to observe or interfere
-with one another like clocks, random number generators, and regular
-expressions.
+with one another like regular expressions and error stacks.
```

Comment:
Long sentence there. How about shortening/breaking it up into:
"`lockdown()` also **tames** some accessible objects that can observe or interfere with each other. These include clocks, random number generators, and regular expressions."